ted2zim/multi/scraper.py

```diff
@@ -60,9 +61,12 @@ def compute_format(item, fmt, mode):
 
     @staticmethod
     def get_playlist_slug(item):
-        return requests.get(f"https://www.ted.com/playlists/{item}").url.replace(
-            f"https://www.ted.com/playlists/{item}/", ""
-        )
+        for attempt in range(5):
+            resp = requests.get(f"https://www.ted.com/playlists/{item}")
```

Comment:
We should add `allow_redirects=True` here. It's there by default but that's what we rely on to get the slug so it would be wise to explicit it.
I think a small comment indicating that we're getting the slug from the final URL after the partial URL is redirected would help.